### PR TITLE
[Wait for #1081] [Loader] Use ini interpreter for the graph

### PR DESCRIPTION
**Pull request by zhoonit**
- [Loader] Use ini interpreter for the graph 

```
This patch substitues graph parsing part to use interpreter.

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

### CODE REVIEW

File: nntrainer/models/model_loader.cpp [hunk 1]
```diff
@@ -478,63 +473,29 @@ int ModelLoader::loadFromIni(std::string ini_file, NeuralNetwork &model,
...
+      std::make_unique<nntrainer::IniGraphInterpreter>(app_context,
+                                                       path_resolver);
+    model.model_graph = *ini_interpreter->deserialize(ini_file);
+  } catch (std::exception &e) {
+    ml_logd("failed to load graph, reason: %s ", e.what());
+    status = ML_ERROR_INVALID_PARAMETER;
+    goto end;
   }
-  ml_logd("parsing ini finished");
+  ml_logd("parsing graph finished");
```
Comment (dongju-chae) [inline nntrainer/models/model_loader.cpp:491]: Well, I think line 491- 496 can be moved to try block, and you may remove `end`.

File: nntrainer/models/model_loader.cpp [hunk 2]
```diff
@@ -478,63 +473,29 @@ int ModelLoader::loadFromIni(std::string ini_file, NeuralNetwork &model,
...
-    status = model.addLayer(layer);
-    NN_INI_RETURN_STATUS();
+  ml_logd("parsing graph started");
+  try {
+    std::unique_ptr<GraphInterpreter> ini_interpreter =
+      std::make_unique<nntrainer::IniGraphInterpreter>(app_context,
+                                                       path_resolver);
+    model.model_graph = *ini_interpreter->deserialize(ini_file);
+  } catch (std::exception &e) {
+    ml_logd("failed to load graph, reason: %s ", e.what());
```
Comment (kparichay) [inline nntrainer/models/model_loader.cpp:487]: logd -> loge?

File: nntrainer/models/model_loader.cpp (hunk 1 above)
Comment (zhoonit) [reply nntrainer/models/model_loader.cpp:491]: Done :)

File: nntrainer/models/model_loader.cpp (hunk 2 above)
Comment (zhoonit) [reply nntrainer/models/model_loader.cpp:487]: updated

**Review by kparichay — APPROVED** [review]
LGTM

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**